cfpb/consumerfinance.gov
# i18n: Add "In this section" and "Close" translations

## Changes

- i18n: Add "In this section" and "Close" translations


## How to test this PR

1. Visit a page like http://localhost:8000/language/ar/coronavirus/mortgage-and-housing-assistance/help-for-homeowners/protections-for-reverse-mortgage-borrowers/ and resize to mobile and see that the secondary nav text is translated.
